server/data/api/crmReportApiClient.ts

```diff
@@ -26,6 +26,27 @@ export default class CrmReportApiClient {
     })
   }
 
+  async getProviderCrmReport(crmReportRequest: CrmReportRequest): Promise<superagent.Response> {
+    const { crmType, decisionFromDate, decisionToDate, providerAccount } = crmReportRequest
+
+    if (!providerAccount) {
```

Comment:
We do not need this if statement here, as the UI validation will stop this. Also the BE API will also validate that this parameter is not null.